runtime/kernel/device/tlkm_control.c

```diff
@@ -50,6 +50,7 @@ static int init_miscdev(struct tlkm_control *pctl)
 	pctl->miscdev.minor = MISC_DYNAMIC_MINOR;
 	pctl->miscdev.name = kstrdup(fn, GFP_KERNEL);
 	pctl->miscdev.fops = &_tlkm_control_fops;
+	pctl->miscdev.mode = 0666;
```

Comment:
Is this required for the simulation? If it is just for convenience, please leave it out of this PR.

Reply:
Just convenience